content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -182,7 +182,7 @@ The .NET Tracer depends on the host operating system, .NET runtime, certain .NET
 The .NET Tracer practices [semantic versioning][11].
 Version updates imply the following changes to runtime support:
 
-  - **Major version updates** (for example `1.0.0` to `2.0.0`) may change support for any runtime from [Beta](#support-beta)/[GA](#support-ga) to [Maintenance](#support-maintenance)/[EOL](#support-eol).
+  - **Major version updates** (for example `1.0.0` to `2.0.0`) may change support for any runtime from [Preview](#support-beta)/[GA](#support-ga) to [Maintenance](#support-maintenance)/[EOL](#support-eol).
```

Comment:
Great point! I agree it is worth updating the anchors since they display prominently in the URL. I forgot about that when I was going through this content initially. I'll fix that!